portfolio/src/main/webapp/comments.html

```diff
@@ -22,25 +22,33 @@
     <div  id="content" class="fade-in">
 
       <form class="center" action="/new-comment" method="POST">
-        Leave a comment!
+        <label for="comment">Leave a comment!</label>
         <br>
         <textarea name="comment" rows="3" required></textarea>
         <br>
+        <label for="name" hidden="true">Your Name (Optional)</label>
         <textarea name="name" type="text" rows="1" placeholder="Your Name (Optional)"></textarea>
         <br>
         <input type="submit" class="btn btn-outline-secondary" />
       </form>
       
       <br>
-      <h6>Number of Comments to Display</h6>
-      <select id="max-num-comments" name="max-comments" min="0" class="custom-select" onchange="loadComments()">
-        <option selected value="all">All</option>
+      <label for="max-num-comments">Number of Comments to Display</id>
```

Comment:
This should be a close label tag, not id

Reply:
Oops